src/OrchardCore.Modules/OrchardCore.Navigation/PagerShapes.cs

```diff
@@ -170,7 +175,8 @@ public PagerShapes(IStringLocalizer<PagerShapes> localizer)
             object GapText,
             bool ShowNext,
             string ItemTagName,
-            IDictionary<string, string> ItemAttributes
+            IDictionary<string, string> ItemAttributes,
+            bool LinkIcons
```

Comment:
`Bool LinkIcons = true` here never worked. But that would have been the appropriate thing to do to have it as default without requiring to add it everywhere.